### DISCUSSION TITLE
Proposal: Disable SFs if SF page turned off

### DISCUSSION THREAD

**Discussion by pfeerick** [EdgeTX/edgetx#4093] [Ideas]
Following #3484, GFs are disabled if the GF page is disabled, which to me makes perfect sense... disable the page, disable its functionality. 

It would therefore seem logical that the same would happen with SFs. This is also in line with the long term goal that was indicated in #1692 that the *associated features* would also be disabled... i.e. if you disable GVs, it's not present in any of the tabs. If you disable HELI, the heli mixer sources go away, etc.

**Comment by philmoz** [comment]
Make sense to me. Should have picked that up in 3484.
